### PR TITLE
Drop EL8 from Pulpcore Nightly

### PR DESCRIPTION
**Pull request by Odilhao**

### CODE REVIEW

File: package_manifest.yaml [hunk 1]
```diff
@@ -16,20 +16,11 @@ packages:
     copr_projects:
       - "{{ hostvars['pulpcore-copr'] }}"
     koji_tags:
```
Comment (ekohl) [inline package_manifest.yaml]: I don't think koji_tags is used at all anymore. Did we forget to clean this up previously?

File: package_manifest.yaml [hunk 2]
```diff
@@ -16,20 +16,11 @@ packages:
...
-      - name: pulpcore-{{ pulpcore_version }}-el8
-        dist: '.el8'
       - name: pulpcore-{{ pulpcore_version }}-el9
         dist: '.el9'
     repoclosure_target_repos:
-      el8:
-        - "el8-pulpcore-{{ pulpcore_version }}-staging"
       el9:
         - "el9-pulpcore-{{ pulpcore_version }}-staging"
-    repoclosure_lookaside_repos:
```
Comment (ekohl) [inline package_manifest.yaml:28]: You should have kept this line in

File: package_manifest.yaml [hunk 3]
```diff
@@ -47,7 +38,6 @@ copr_projects:
     core_modules:
       - 'ruby:2.7'
```
Comment (ekohl) [inline package_manifest.yaml]: This looks unused now

**Review by ekohl — CHANGES_REQUESTED** [review]

File: package_manifest.yaml (hunk 2 above)
Comment (Odilhao) [reply package_manifest.yaml:28]: Yes, this one was left behind

File: package_manifest.yaml [hunk 4]
```diff
@@ -15,21 +15,10 @@ packages:
     build_package_use_koji_build: true
```
Comment (ekohl) [inline package_manifest.yaml:15]: Is this also still relevant? I think this was made redundant when we switched to copr.

**Review by ekohl — APPROVED** [review]
:+1: once we've dropped EL8 from Foreman

**Pull request merged**